### PR TITLE
Github Apps Documentation

### PR DESCRIPTION
**Pull request by RakshithaK06**
Story details: https://app.clubhouse.io/kintohub/story/578

Updated "Import URL" step.

### CODE REVIEW

File: docs/anatomy/anatomy-github-apps.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+2. Click on the **Create Service** button at the top right corner
+3. Choose a service from the list of services provided
+4. Click on the **Connect Github** button
+
+![Connect Github btn](/img/anatomy/connect-github-btn.png)
+:::note
+If your GitHub account has multiple organisations linked to it, you will be provided with an option to choose the appropriate organisation and access the repositories under that organisation on KintoHub.
+:::
+5. To select only preferred repositories, enable the option **Only select repositories** and then make your selection
+6. Click on he **Install & Autorize** button
```
Comment (Disturbing) [inline docs/anatomy/anatomy-github-apps.md:21]: action - Autorize -> Authorize? Make sure you have a spell checker.

File: docs/anatomy/anatomy-github-apps.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+2. Click on the **environment** name displayed in the top left corner and select appropriate environment
+3. Click on the **Create Service** button
+4. Choose a service from the list of services provided
+5. Click on the **Edit Configuration** link displayed in the `GitHub` tab
+6. Sroll down to the **Danger Zone** section
+7. Click on the **Uninstall** button
+
+![Uninstall](/img/anatomy/Uninstall.png)
+
+The repositories in your Github account will no longer be accessible on KintoHub.
```
Comment (Disturbing) [inline docs/anatomy/anatomy-github-apps.md:68]: action - maybe make it more clear that your services will still be on kintohub, but will no longer be able to build and deploy new releases.  All of this on top of that their github account will no longer be accessible by KintoHub

File: sidebars.js [hunk 3]
```diff
@@ -6,6 +6,7 @@ module.exports = {
     ],
     "Anatomy by Feature": [
       'anatomy/environment',
+      'anatomy/github-apps',
```
Comment (Disturbing) [inline sidebars.js:9]: Action - Let's add this as a sub-section to Repository. Check out examples to see how i did subsections.

File: docs/anatomy/anatomy-repository.md [hunk 4]
```diff
@@ -13,10 +13,11 @@ Repositories do not have any requirements to be compatible with KintoHub. You ca
...
-3. Enter in your **Git Repository URL**
-4. Enter in your **Branch** you wish to build and deploy
-5. Optionally add an **Access Token** to your repository if it is private.
+2. Click on the **Create Service** button at the top right
+3. Click on the **Import URL** tab
+4. Enter in your **Git Repository URL**
+5. Enter in your **Branch** you wish to build and deploy
+6. Optionally add an **Access Token** to your repository if it is private.
 
 ## Edit Existing Repo
```
Comment (Disturbing) [inline docs/anatomy/anatomy-repository.md:22]: action - in this repository thing. Maybe at the top of the page after describing the feature, "Looking to connect your repoisotires via Github Apps. Read more here. (link to github apps stuff)

**Review by Disturbing — CHANGES_REQUESTED** [review]

File: docs/anatomy/anatomy-github-apps.md (hunk 1 above)
Comment (RakshithaK06) [reply docs/anatomy/anatomy-github-apps.md:21]: changed Autorize -> Authorize

Comment (RakshithaK06) [reply docs/anatomy/anatomy-github-apps.md]: Changed Autorize -> Authorize.

File: sidebars.js (hunk 3 above)
Comment (RakshithaK06) [reply sidebars.js]: Done. Created a new sub-section to Repository.

File: docs/anatomy/anatomy-repository.md (hunk 4 above)
Comment (RakshithaK06) [reply docs/anatomy/anatomy-repository.md:24]: Added a link to Github Apps.

File: docs/anatomy/anatomy-github-apps.md (hunk 2 above)
Comment (RakshithaK06) [reply docs/anatomy/anatomy-github-apps.md]: Done.

**Review by Disturbing — APPROVED** [review]

**Pull request merged**